recipes/napari-tyssue/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "napari-tyssue" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-tyssue-{{ version }}.tar.gz
+  sha256: 24f453e2a418062770b6bc932f560481b0e9992499a0b0b031d9d036b39d39ff
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8,<3.10
+  run:
+    - imageio-ffmpeg
+    - invagination ==0.0.2
```

Comment:
@kephale any chance we can change this to `invagination <=0.0.2` or `invagination >=0.0.2` ?

Reply:
`invagination >=0.0.2` would be my choice